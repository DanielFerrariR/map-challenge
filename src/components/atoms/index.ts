export { default as Box } from './box'
export { default as Paper } from './paper'
export * from './paper'
export { default as Typography } from './typography'
export { default as IconButton } from './icon_button'
export { default as Tooltip } from './tooltip'
export { default as Modal } from './modal'
export { default as ListItemText } from './list_item_text'
export { default as ListItemSecondaryAction } from './list_item_secondary_action'
export { default as Fade } from './fade'
export { default as Backdrop } from './backdrop'
export { default as CircularProgress } from './circular_progress'
export { default as List } from './list'
export { default as ListItem } from './list_item'
export { default as InputBase } from './input_base'
export { default as Button } from './button'
