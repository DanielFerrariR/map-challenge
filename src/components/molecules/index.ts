export { default as MapSearchInput } from './map_search_input'
export { default as PlacesModal } from './places_modal'
export { default as Modal } from './modal'
