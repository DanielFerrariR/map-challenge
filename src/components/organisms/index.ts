// eslint-disable-next-line import/prefer-default-export
export { default as MapContainer } from './map_container'
