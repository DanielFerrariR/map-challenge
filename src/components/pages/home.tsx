import React from 'react'
import { Box } from 'src/components/atoms'
import { MapContainer } from 'src/components/organisms'

const Home: React.FC = () => {
  return (
    <Box height={1}>
      <MapContainer />
    </Box>
  )
}

export default Home
