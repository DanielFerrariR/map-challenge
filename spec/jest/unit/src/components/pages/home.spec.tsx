import React from 'react'
import { Home } from 'src/components/pages'
import { render } from 'spec/jest/config/test_utils'
import { axe } from 'jest-axe'

describe('testing Home component', () => {
  test('should snapshot works', () => {
    const { container } = render(<Home />)

    expect(container).toMatchSnapshot()
  })

  test('should the page be accessible', async () => {
    const { container } = render(<Home />)
    const results = await axe(container)

    expect(results).toHaveNoViolations()
  })
})
