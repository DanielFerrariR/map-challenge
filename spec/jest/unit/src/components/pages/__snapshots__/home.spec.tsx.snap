// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Home component should snapshot works 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root Component-root Component-root"
  />
</div>
`;
